src/ray/raylet/main.cc

```diff
@@ -39,11 +40,8 @@ int main(int argc, char *argv[]) {
       RayConfig::instance().num_workers_per_process();
   // Use a default worker that can execute empty tasks with dependencies.
 
-  std::stringstream worker_command_stream(worker_command);
-  std::string token;
-  while (getline(worker_command_stream, token, ' ')) {
-    node_manager_config.worker_command.push_back(token);
-  }
+  boost::split(node_manager_config.worker_command, worker_command,
+               boost::is_any_of(" "), boost::token_compress_on);
```

Comment:
Let's revert this. I think we want to minimize boost dependencies, so currently we just use `boost::asio`.

cc @pcmoritz

Reply:
@robertnishihara 
This small change have fixed a bug：
If we pass a command string like "a&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;b"(note:more than 1 space between a and b), this code will parse it into an array which contains `('a',' ', ' ',' ', ' ', 'b' )`. This will emit some errors when `startprocess(`('a',' ', ' ',' ', ' ', 'b' )`)` because of some useless spaces.
Therefore, we should compress some useless spaces when parse command string.

If we want to minimize boost dep, I will do this without it.

Reply:
Oh that makes sense, good catch :) Still let's do this without boost.